packages/cli/generators/app/templates/src/controllers/ping.controller.ts.ejs

```diff
@@ -16,10 +16,10 @@ const PING_RESPONSE: ResponseObject = {
           url: {type: 'string'},
           headers: {
             type: 'object',
-            patternProperties: {
-              '^.*$': {type: 'string'},
+            properties: {
```

Comment:
nitpick: I think we can remove `properties`.

Reply:
No, we can't.  it accepts the removal of headers completely, but it needs the properties: .   what we can remove is the additionalProperties or the headers.